packages/SystemsThinkingWorkbench-Core.package/STWTransferConnection.class/instance/input.st

```diff
@@ -0,0 +1,3 @@
+accessing
+input
+	^ input
```

Comment:
Let me rephrase that:

Maybe it is not lacking documentation, since good code documents itself.

I am not sure on the naming of this instance variable.
Input sound passive on behalf of the connection (as in it receives input) but our system is modelled in a way where the connection is the active object in the simulation.
Maybe we could name it `intake`(as in it takes some (for lack of a better word) blips from it's start node).